projects/netgrif-components/src/lib/panel/panel-item/panel-item.component.html

```diff
@@ -0,0 +1,19 @@
+<span class='text-column-margin' fxLayoutAlign='start center'>
+    <mat-icon class='panel-icon panel-primary-icon' *ngIf='index === 0'>
```

Comment:
the `panel-primary-icon` class is missing from the scss file of this component

Reply:
It is present in panel.theme.scss file that is a wrapper theme file for whole panel component. It is not even present in the task-panel.component.scss nor in case-panel.component.ts, that served as old solution for this panel featured values. Is it ok?

Reply:
is the panel theme then imported into this component somehow? Does the class do anything in the browser?